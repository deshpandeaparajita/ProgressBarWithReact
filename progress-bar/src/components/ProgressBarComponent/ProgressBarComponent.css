
.progress-bar {
    width: 350px;
    height: 35px;
    border-radius: 50px;
    border: 2px solid #666;
    margin-bottom: 40px;
}
