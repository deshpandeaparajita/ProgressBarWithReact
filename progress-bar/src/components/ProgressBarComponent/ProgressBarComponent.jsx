import React from 'react';

export const ProgressBarComponent = (props) => {


    return(<div>This is for Progress bar component</div>);
};

export default ProgressBarComponent;