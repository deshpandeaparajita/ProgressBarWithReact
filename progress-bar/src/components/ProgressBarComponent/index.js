export {default as ProgressBarComponent} from './ProgressBarComponent';