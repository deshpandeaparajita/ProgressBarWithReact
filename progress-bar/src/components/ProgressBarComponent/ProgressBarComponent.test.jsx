import React from 'react';
import { shallow } from 'enzyme';
import ProgressBarComponent from './ProgressBarComponent';



describe('ProgressBarComponent', () => {
    it('should match snapshot', () => {
        const component = shallow(<ProgressBarComponent />);
        expect(component).toMatchSnapshot();
    });
});