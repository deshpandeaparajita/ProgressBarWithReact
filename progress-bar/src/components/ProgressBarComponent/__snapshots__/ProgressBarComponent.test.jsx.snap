// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBarComponent should match snapshot 1`] = `ShallowWrapper {}`;
